app/themes/baseTheme.js

```diff
@@ -45,6 +45,8 @@ export const fontWeights = {
   black: 900,
 };
 
+export const lineHeights = [1, 1.25, 1.33, 1.5, 1.75, 2];
```

Comment:
Should this also be an exported property on the default export?

Reply:
Oh! :) good call

Reply:
Done.  Merging now